src/layouts/components/Homepage/InfobarBody.tsx

```diff
@@ -0,0 +1,100 @@
+import { FormControl } from "@chakra-ui/react"
+import {
+  Button,
+  FormErrorMessage,
+  FormLabel,
+  Input,
+  Textarea,
+} from "@opengovsg/design-system-react"
+
+import { Editable } from "../Editable"
+
+interface InfobarInputs {
+  title: string
+  subtitle: string
+  description: string
+  button: string
+  url: string
+}
+
+interface InfobarBodyProps extends InfobarInputs {
+  index: number
+  onClick: () => void
+  onChange: () => void
+  errors: InfobarInputs
+}
+
+export const InfobarBody = ({
+  title,
+  subtitle,
+  description,
+  button,
+  url,
+  index,
+  onClick,
+  onChange,
+  errors,
+}: InfobarBodyProps) => (
+  <Editable.Section>
+    <FormControl isRequired isInvalid={!!errors.subtitle}>
+      <FormLabel>Subtitle</FormLabel>
+      <Input
+        placeholder="This subtitle appears above the title"
+        id={`section-${index}-infobar-subtitle`}
+        value={subtitle}
+        onChange={onChange}
```

Comment:
(Info, no change required)
New sections are created with default text in most fields, so we don't actually see the placeholder value for most of these components unless the user completely removes the text - just checking that design is aware!